doc/whatsnew/fragments/4655.bugfix

```diff
@@ -0,0 +1,3 @@
+Any assertion on a populated tuple will now receive a `assert-on-tuple` warning.
```

Comment:
```suggestion
Any assertion on a populated tuple will now receive a ``assert-on-tuple`` warning.
```

With the double backticks it will appear as a code snippet. Single renders as italic.
https://pylint.pycqa.org/en/latest/whatsnew/2/2.16/index.html